model/job_test.go

```diff
@@ -24,13 +24,16 @@ func TestJobInfoOk(t *testing.T) {
 	assert.Nil(err)
 
 	assert.Equal(1, len(release.Jobs))
+	const ntpd_fingerprint = "9c168f583bc177f91e6ef6ef1eab1b4550b78b1e"
+	const ntpd_version = ntpd_fingerprint
+	const ntpd_sha1 = "aab8da0094ac318f790ca40c53f7a5f4e137f841"
 
 	assert.Equal("ntpd", release.Jobs[0].Name)
-	assert.Equal("f1f3607917dfd9d64580f3a97d71b60c2545c51a", release.Jobs[0].Version)
-	assert.Equal("f1f3607917dfd9d64580f3a97d71b60c2545c51a", release.Jobs[0].Fingerprint)
-	assert.Equal("cf1722e891564d1f667c2218d6685679fe149591", release.Jobs[0].SHA1)
+	assert.Equal(ntpd_fingerprint, release.Jobs[0].Version)
```

Comment:
The linter doesn't like these variable names (and even suggests how to rename them!).
https://concourse.from-the.cloud/builds/9958